CHANGES

```diff
@@ -1,6 +1,15 @@
 Changes beyond 8.7
 ==================
 
+OCaml
+
+- Users can pass specific flags to the OCaml optimizing compiler by
+  -using the flambda-opts configure-time option.
+
+  Beware that compiling Coq with a flambda-enabled compiler is
+  experimental and may require large amounts of RAM and CPU, see
+  INSTALL for more details.
+
```

Comment:
I have no strong feelings against having this PR shipped into 8.7 (although some more testing in master before shipping it in a stable release couldn't have harmed), but if you really want this in 8.7, please move this entry to a new section:

```
Changes from 8.7+beta2 to 8.7.0
===============================
```

Reply:
Ok. Testing of this PR is a complicated matter as it requires a different toolchain plus some expertise on the builder, it is surprising few people is interested on this, given how critical Coq performance is for many.

Reply:
> it is surprising few people is interested on this, given how critical Coq performance is for many.

Not saying anything does not mean not being interested in the performance gains this gives :smile: